src/jailer/src/env.rs

```diff
@@ -256,7 +273,7 @@ impl Env {
 
         // This will not only create necessary directories, but will also change ownership
         // for all of them.
-        self.setup_jailed_folders()?;
+        self.setup_jailed_folders(FOLDER_HIERARCHY.to_vec())?;
```

Comment:
nit: this is slightly slower

Reply:
But unit testable :grinning: To reiterate, I'm thinking some more on how to rework this section to get the best of both worlds